poker.py

```diff
@@ -1 +1,147 @@
+user_cards = []
+user_money = 50
+player1_cards = []
+global player1_money
+player1_money = 50
+player2_cards = []
+global player2_money
+player2_money = 50
+player3_cards = []
+global player3_money
+player3_money = 50
+betting_choices = [5,10,15]
+welcome = """Welcome to Python Poker! here you will play a text-based version of 5-card poker, written in python code!
+There will be three other players in the game. You will now receive your five cards."""
+choices = ['calling', 'raising', 'folding']
+def play():
+    global player1_money
+    global player2_money
+    global player3_money
+    user_cards = []
+    player1_cards = []
+    player1_money = 50
+    player2_cards = []
+    player2_money = 50
+    player3_cards = []
+    player3_money = 50
+    print welcome
+    random.shuffle(deck)
+    user_cards.append(random.choice(deck))
+    user_cards.append(random.choice(deck))
+    user_cards.append(random.choice(deck))
+    user_cards.append(random.choice(deck))
+    user_cards.append(random.choice(deck))
+    player1_cards.append(random.choice(deck))
+    player1_cards.append(random.choice(deck))
+    player1_cards.append(random.choice(deck))
+    player1_cards.append(random.choice(deck))
+    player1_cards.append(random.choice(deck))
+    player2_cards.append(random.choice(deck))
+    player2_cards.append(random.choice(deck))
+    player2_cards.append(random.choice(deck))
+    player2_cards.append(random.choice(deck))
+    player2_cards.append(random.choice(deck))
+    player3_cards.append(random.choice(deck))
+    player3_cards.append(random.choice(deck))
+    player3_cards.append(random.choice(deck))
+    player3_cards.append(random.choice(deck))
+    player3_cards.append(random.choice(deck))
+    player1_money -= 5
+    player2_money -= 5
+    player3_money -= 5
+    print 'You have paid your ante, now you can receive your cards.'
+    print user_cards
+    print 'The first betting round will now commence. You have $45 to bet. The ante was 5.'
+    betting_round()
+def choosing_betting(player_money, player):
+    global player1_money
+    global player2_money
+    global player3_money
+    global last_call
+    global pot
+    last_call = 5
+    pot = 0
+    pot += 20
+    if random.choice(choices) == 'calling':
```

Comment:
you may actually never call this if the random choice doesnt equal your if statement ...